### PR TITLE
[refactor] remove caze gems with dependency upgrades

### PR DESCRIPTION
**Pull request by jordansgrant**
Addressing: https://github.com/pusher/push-notifications-ruby/issues/20

### CODE REVIEW

File: spec/pusher/push_notifications/generate_token_spec.rb [hunk 1]
```diff
@@ -3,51 +3,49 @@
 require 'spec_helper'
 require 'jwt'
 
-RSpec.describe Pusher::PushNotifications::UseCases::GenerateToken do
-  subject(:use_case) { described_class.new(user: user) }
+RSpec.describe Pusher::PushNotifications, '.generate_token' do
+  def generate_token
+    described_class.generate_token(user: user)
```
Comment (jordansgrant) [inline spec/pusher/push_notifications/generate_token_spec.rb:8]: Test use cases through the `PushNotifications` module in order to exercise the delegation

**Review by elverkilde — APPROVED** [review]

**Comment by jordansgrant** [comment]
@elverkilde Thanks for the review! Just checking if there is a timeline for getting this merged and released?

**Comment by elverkilde** [comment]
@jordansgrant no problem, thank you for your contribution! I am aiming to get it released this week, might just need a second pair of eyes.

**Review by GeorgeRDBuckley — APPROVED** [review]
This looks good to me 👍

**Pull request merged**